### PR TITLE
Household member validations

### PR DESCRIPTION
**Pull request by baarkerlounger**
https://digital.dclg.gov.uk/jira/browse/CLDC-1066

- [x] Adds validation that if age is > 16 the economic status can't be child under 16
- [x] For compound validations in the household sections adds error messages to all fields that could trigger it
- [x] Updates the ecstat values to match the data dictionary (attached in Jira)
- [x] Updates the allowed income ranges to match the data dictionary (attached in Jira)

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -152,15 +152,24 @@ en:
       age:
```
Comment (baarkerlounger) [inline config/locales/en.yml:152]: @Dan13L3 could you check these messages please?

File: app/models/validations/household_validations.rb [hunk 2]
```diff
@@ -131,11 +131,17 @@ def validate_person_age_matches_economic_status(record, person_num)
...
+      record.errors.add "age#{person_num}", I18n.t("validations.household.age.retired_over_70", person_num:)
     end
-    if age < 16 && economic_status != 8
+    if age < 16 && economic_status != 9
       record.errors.add "ecstat#{person_num}", I18n.t("validations.household.ecstat.child_under_16", person_num:)
+      record.errors.add "age#{person_num}", I18n.t("validations.household.age.child_under_16", person_num:)
+    end
+    if economic_status == 9 && age > 16
+      record.errors.add "ecstat#{person_num}", I18n.t("validations.household.ecstat.child_over_16", person_num:)
+      record.errors.add "age#{person_num}", I18n.t("validations.household.age.child_over_16", person_num:)
```
Comment (dushan-madetech) [inline app/models/validations/household_validations.rb]: I wonder if we could have some helper methods here to clarify what the economic status integers actually mean

**Review by dushan-madetech — APPROVED** [review]
Single comment but otherwise looks good to me 👍

File: app/models/validations/household_validations.rb (hunk 2 above)
Comment (baarkerlounger) [reply app/models/validations/household_validations.rb]: Right you are

**Pull request merged**